### PR TITLE
Host throws assembly scanning errors on startup

### PR DESCRIPTION
**Pull request by andreasohlund**
This fixes the bug but we should make sure we can repro with a unit test before merging this.

https://microsoft.github.io/AzureTipsAndTricks/blog/tip196.html and https://stackoverflow.com/questions/59743560/how-to-unit-test-azure-functions-in-visual-studio show how to use func.exe but there might be other better ways

### CODE REVIEW

File: src/NServiceBus.AzureFunctions.InProcess.ServiceBus/InProcessFunctionEndpoint.cs [hunk 1]
```diff
@@ -239,7 +239,9 @@ static async Task SafeCompleteMessageAsync(ServiceBusMessageActions messageActio
             "Grpc.Net.ClientFactory.dll",
             "Azure.Identity.dll",
             "Microsoft.Extensions.Azure.dll",
-            "NServiceBus.Extensions.DependencyInjection.dll"
+            "NServiceBus.Extensions.DependencyInjection.dll",
```
Comment (andreasohlund) [inline src/NServiceBus.AzureFunctions.InProcess.ServiceBus/InProcessFunctionEndpoint.cs:242]: This feels like a never-ending story of patches as MS adds more references to the azure host but not sure how to make this better

Comment (DavidBoike) [reply src/NServiceBus.AzureFunctions.InProcess.ServiceBus/InProcessFunctionEndpoint.cs:242]: Really? That's all? Would this work in InternalAutomation if I added these excludes in the NSB config?

For CI you can [install the functions core tools with automated tools](https://github.com/Azure/azure-functions-core-tools#installing)

Comment (andreasohlund) [reply src/NServiceBus.AzureFunctions.InProcess.ServiceBus/InProcessFunctionEndpoint.cs:242]: Yeah, that should work

Thanks for the link👍

**Review by SzymonPobiega — APPROVED** [review]

**Comment by andreasohlund** [comment]
@SzymonPobiega @WojcikMike ready for final review

**Review by SzymonPobiega — APPROVED** [review]

**Pull request merged**